core/templates/dev/head/pages/topic_viewer/topic_viewer.html

```diff
@@ -0,0 +1,134 @@
+{% extends 'pages/base.html' %}
+
+{% block maintitle %}
+  {{ topic_name }} - Oppia
+{% endblock maintitle %}
+
+{% block navbar_breadcrumb %}
+  <topics-viewer-navbar-breadcrumb>
+  </topics-viewer-navbar-breadcrumb>
+{% endblock navbar_breadcrumb %}
+
+{% block content %}
+  <background-banner></background-banner>
+  <div ng-controller="TopicViewer" class="oppia-topic-viewer-container" style="position: relative;">
+    <div class="oppia-topic-name">
+      {{ topic_name }}
```

Comment:
We're trying to get rid of Jinja. Here and elsewhere, could you please use angular interpolation instead?

I think the collection editor provides an example of how to do this properly.

Reply:
Oh yes! I must have missed it. Thanks! Will remove with rest of the changes (that will be told in the changes, if any)